app/controllers/admin/pdfs_controller.rb

```diff
@@ -0,0 +1,19 @@
+class Admin::PdfsController < ApplicationController
+  def new
+  end
+
+  def create
+    pdf = Pdf.new(pdf_params)
+    if pdf.save
```

Comment:
I think you will... it's unlikely my estimate for our routes is right on the first try